commons/all/src/main/java/org/infinispan/commons/api/CacheContainerAdmin.java

```diff
@@ -19,21 +22,25 @@ public interface CacheContainerAdmin<C extends CacheContainerAdmin, A extends Ba
     * @author Tristan Tarrant
     * @since 9.2
     */
+   @ProtoTypeId(ProtoStreamTypeIds.CACHE_CONTAINER_ADMIN_FLAG)
    enum AdminFlag {
```

Comment:
It can be annotated with `@Proto` and remove `@ProtoEnumValue`. It looks cleaner to me. Just my 2 cents.

Reply:
I'll update. I suspect there will be a few more instances where this is the case, the joys of converting some of the code a long time ago before `@Proto` was available.